torch/distributions.py

```diff
@@ -197,6 +198,26 @@ def log_prob(self, value):
         return -((value - self.mean) ** 2) / (2 * var) - log_std - math.log(math.sqrt(2 * math.pi))
 
 
+class _StandardGamma(Function):
```

Comment:
This is an old-style autograd function. You should write it as shown [in the docs](http://pytorch.org/docs/master/notes/extending.html#extending-torch-autograd).

Reply:
I wasn't able to store the gradient via `ctx.save_for_backward(grad)`. Is there a new-style way to save an intermediate that is neither an input nor an output?

Reply:
You can't do that because that can only be done with inputs/outputs. Just do `ctx.grad = grad`.

Reply:
Done.